src/components/Header/components/PeopleSearch/index.js

```diff
@@ -110,6 +111,10 @@ export default class GordonPeopleSearch extends Component {
       if (suggestionIndex === -1) suggestionIndex = suggestionList.length - 1;
       this.setState({ suggestionIndex });
     }
+    if (key === 'Backspace') { // Bug fixed where suggestions are remembered after backspacing
+      const suggestions = [];
```

Comment:
Should this distinguish whether the resulting search string is still long enough?  Maybe I just don't understand the code well enough, but this looks like it will get rid of suggestions if I type 5 letters and then 1 backspace (leaving 4 letters, which should be enough to get suggestions).  Does clearing suggestions cause them to be re-fetched?

Reply:
So, I did some manual testing and it still re-fetches suggestions. I think it just has to do with clearing the suggestions on the key event and then the query repopulates the array, updates suggestions again, and shows them.